packages/rulesets/generated/spectral/az-arm.js

```diff
@@ -3354,7 +3354,6 @@ const ruleset = {
             rpcGuidelineCode: "RPC-Policy-V1-03",
             description: "Properties with type:object that don't reference a model definition are not allowed. ARM doesn't allow generic type definitions as this leads to bad customer experience.",
```

Comment:
I agree with this one, but we do not have a similar rule inside TypeSpec.  Filed this issue to resolve: https://github.com/Azure/typespec-azure/issues/1426